code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -149,9 +149,9 @@
 	singular_name = "floor tile"
 	desc = "Those could work as a pretty decent throwing weapon."
 	icon_state = "tile"
-	force = 6
+	force = 12
```

Comment:
i wonder how many other "30%" stealth buffs are in here he didnt tell us about :^)